x-pack/plugins/ml/public/application/trained_models/models_management/test_models/models/text_embedding/text_embedding_inference.ts

```diff
@@ -37,6 +39,16 @@ export class TextEmbeddingInference extends InferenceBase<TextEmbeddingResponse>
     }),
   ];
 
+  constructor(
+    trainedModelsApi: ReturnType<typeof trainedModelsApiProvider>,
+    model: estypes.MlTrainedModelConfig,
+    inputType: INPUT_TYPE
+  ) {
+    super(trainedModelsApi, model, inputType);
+
+    this.initializeValidators();
```

Comment:
perhaps we can have a generic `initialize` method that calls `initializeValidators`? just in case there will be more logic on initialisation

Reply:
I've added the list of additional validators to `initializeValidators` as suggested, which means if we have an extra `initialize` function, that too would need to take these additional validators to pass on.
I think just having `initializeValidators` is good for now.